src/signalrclient/default_websocket_client.cpp

```diff
@@ -69,7 +69,9 @@ namespace signalr
 
         if (transfer_format == signalr::transfer_format::binary)
         {
-            throw signalr_exception("binary isn't supported currently");
+            concurrency::streams::producer_consumer_buffer<uint8_t> b;
```

Comment:
This looks like a pipe.

Reply:
It's certainly a more descriptive name 😝